ecl/dedicated_hypervisor/exceptions.py

```diff
@@ -6,7 +6,21 @@
 
 
 class HttpException(exceptions.HttpException):
-    pass
+
+    def _get_exception_message(self, message=None):
+        try:
+            content = json.loads(self.response._content.decode('utf-8'))
+
+            # vCenter error case.
+            key = list(content.key())[0]
+            code = content[key]['code']
+            message = content[key]['message']
+            return '[' + code + ']' + message
```

Comment:
```suggestion
            return '[' + str(code) + ']' + message
```
codeは明示的に文字列化する必要があります。

Reply:
ご指摘のとおり修正しました。